std_msgs/String move_group
geometry_msgs/PoseArray target_pose_array
std_msgs/Float32 velocity_scaling_factor
std_msgs/Float64 min_path_completion
std_msgs/Int32 max_attempts
std_msgs/Float32 max_planning_time
---
std_msgs/Bool success
